pkgs/development/haskell-modules/configuration-hackage2nix/main.yaml

```diff
@@ -629,6 +629,7 @@ unsupported-platforms:
   Jazzkell:                                     [ platforms.darwin ] # depends on Euterpea
   jsaddle-hello:                                [ platforms.darwin ] # depends on jsaddle-webkit2gtk
   jsaddle-webkit2gtk:                           [ platforms.darwin ]
+  jsaddle-wkwebview:                            [ platforms.linux  ] # only builds on darwin
```

Comment:
Since we know that only one plattform is supported, I’d recommend using the `supported-platforms` list further below.